release-notes/v3.31.0-release-notes.md

```diff
@@ -0,0 +1,204 @@
+- Export cluster IPs for all IP families [calico 10183](https://github.com/projectcalico/calico/pull/10183) (@nick-oconnor)
+- All Docker images now use standard OCI labels [calico 10181](https://github.com/projectcalico/calico/pull/10181) (@sebhoss)
+- Update Operator Helm values to support custom DNS settings [calico 10117](https://github.com/projectcalico/calico/pull/10117) (@jsturtevant)
+- Performance: Felix now stores its selector index more efficiently resulting in reduced memory usage in large clusters. [calico 10027](https://github.com/projectcalico/calico/pull/10027) (@fasaxc)
+- Updated Calico to use quay.io as the default image registry, replacing docker.io. [calico 10025](https://github.com/projectcalico/calico/pull/10025) (@skoryk-oleksandr)
+- Performance: After failures to program IP sets, the dataplane now does a partial resync of only the affected IP sets. This should reduce the overhead of dealing with a transient failure. [calico 9674](https://github.com/projectcalico/calico/pull/9674) (@fasaxc)
+- New Felix config param natOutgoingExclusions allows for configuring which type of destinations is excluded from being masqueraded. [calico 8961](https://github.com/projectcalico/calico/pull/8961) (@wayne-cheng)
+
+##### OpenStack
+
+- OpenStack: support IPv6 subnets using SLAAC [calico 11027](https://github.com/projectcalico/calico/pull/11027) (@nelljerram)
+- OpenStack QoS handling has been improved to respond immediately to all kinds of QoS changes on the Neutron API.  In particular when rules are added, updated or deleted for a QoS policy that is already in use, and when a QoS policy is added, updated or deleted for a Neutron network that is already in use.  Calico's handling of these changes previously relied on a periodic resync, which could in practice take a long time; now those changes are handled immediately (within a few seconds). [calico 10931](https://github.com/projectcalico/calico/pull/10931) (@nelljerram)
+- Calico for OpenStack's QoS support has been enhanced to cover packet rate burst size configuration.  There are new driver settings (cluster-wide, set in `neutron.conf`) for configuring max and min packet burst sizes in each direction. [calico 10521](https://github.com/projectcalico/calico/pull/10521) (@nelljerram)
+- Calico for OpenStack's QoS support has been enhanced to cover peak rate configuration.  The "max_burst_kbps" field of bandwidth limit rules is honoured as the peak rate, and there are new driver settings (cluster-wide, set in `neutron.conf`) for configuring max and min burst sizes in each direction. [calico 10500](https://github.com/projectcalico/calico/pull/10500) (@nelljerram)
+- Add Ubuntu 24.04 packages for OpenStack [calico 8685](https://github.com/projectcalico/calico/pull/8685) (@danudey)
+
+##### eBPF
+
+- ebpf: BPFKubeProxyHealtzPort allows to change the kube-proxy healthz server port [calico 11152](https://github.com/projectcalico/calico/pull/11152) (@tomastigera)
```

Comment:
@tomastigera is "BPFKubeProxyHealtzPort" a slight typo ("Healtz" vs "Healthz")?

I guess it's too late to fix it up. I guess at least the typo was applied consistently across the configuration option and the documentation for it. 😅

Reply:
```suggestion
- ebpf: BPFKubeProxyHealthzPort allows to change the kube-proxy healthz server port [calico 11152](https://github.com/projectcalico/calico/pull/11152) (@tomastigera)
```